ogr/ogrsf_frmts/openfilegdb/gdalopenfilegdbrasterband.cpp

```diff
@@ -444,9 +444,46 @@ bool OGROpenFileGDBDataSource::OpenRaster(const GDALOpenInfo *poOpenInfo,
         m_adfGeoTransform[5] = -dfResY;
     }
 
-    // Get SRID, and fetch WKT from GDBSpatialRefs table
+    // Get SRID, and fetch WKT from GDBSpatialRefs table (for FileGDB v
```

Comment:
Is this supposed to be?

```suggestion
    // Get SRID, and fetch WKT from GDBSpatialRefs table (for FileGDB v9)
```

Reply:
actually not :-). I've expanded the comment to:
```
    // Two cases:
    // - osDefinition is empty (that is FileGDB v9): find the SRS by looking
    //   at the SRS attached to the RASTER field definition of the .gdbtable
    //   file of the main table of the raster (that is the one without fras_XXX
    //   prefixes)
    // - or osDefinition is not empty (that is FileGDB v10): get SRID from the
    //   "srid" field of the _fras_bnd table, and use that has the key to
    //   lookup the corresponding WKT from the GDBSpatialRefs table.
    //   In some cases srid might be 0 (invalid), then we try to get it from
    //   Definition column of the GDB_Items table, stored in osDefinition
```

Reply:
Thanks!